CONFIGDOC.md

```diff
@@ -266,13 +276,17 @@ node {
 
 MATLAB exits with exit code 0 if the test suite runs successfully without any test failures. Otherwise, MATLAB terminates with a nonzero exit code, which causes the current stage to fail. If you properly react to the resulting MATLAB execution exception, the remaining stages of your Pipeline can still run, and your build can succeed. Otherwise, Jenkins terminates the build in the current stage and marks it as a failure.
 
-The `runMATLABTests` step lets you customize your test run using optional inputs. For example, you can add folders to the MATLAB search path, control which tests to run, and generate various artifacts.
+You can customize the `runMATLABTests` step using optional inputs. For example, you can add folders to the MATLAB search path, control which tests to run, and generate various artifacts.
 
 Input                     | Description    
 ------------------------- | ---------------
 `sourceFolder`            | (Optional) Location of the folder containing source code, relative to the project root folder. The specified folder and its subfolders are added to the top of the MATLAB search path. If you specify `sourceFolder` and then generate a coverage report, the plugin uses only the source code in the specified folder and its subfolders to generate the report.<br/>**Example:** `['source']`<br/>**Example:** `['source/folderA', 'source/folderB']`
 `selectByFolder`          | (Optional) Location of the folder used to select test suite elements, relative to the project root folder. To create a test suite, the plugin uses only the tests in the specified folder and its subfolders.<br/>**Example:** `['test']`<br/>**Example:** `['test/folderA', 'test/folderB']`
 `selectByTag`             | (Optional) Test tag used to select test suite elements. To create a test suite, the plugin uses only the test elements with the specified tag.<br/>**Example:** `'FeatureA'`
+`strict`                  | (Optional) Whether to apply strict checks when running the tests, specified as `false` or `true`. By default, the value is `false`. If you specify a value of `true`, the plugin generates a qualification failure whenever a test issues a warning.
+`useParallel`             | (Optional) Whether to run tests in parallel, specified as `false` or `true`. By default, the value is `false` and tests run in serial. If the test runner configuration is suited for parallelization, you can specify a value of `true` to run tests in parallel. This input requires a Parallel Computing Toolbox&trade; license.
+`outputDetail`            | (Optional) Amount of event detail displayed for the test run, specified as `none`, `terse`, `concise`, `detailed`, or `verbose`. Selecting a value for this input is the same as specifying the `OutputDetail` name-value argument of [`runtests`](https://www.mathworks.com/help/matlab/ref/runtests.html) as that value.

```

Comment:
there is an issue to consider here that  when someone specify the OutPutDetail or Logging level in pipeline then only values supported are as described in document. `Default` as value will throw an warnning and fall backs to the default state of the level which is 4 or detailed. closes issue #224 . per me we can clearly mention this in quick note section. so that we can avoid this as Xbug

Reply:
Added information on the default behavior. Not sure whether we need a note because the users are provided with a finite list of options and should choose from that list.